tests/unit/feature_flags.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+from __future__ import annotations
+
+import copy
+from dataclasses import asdict, is_dataclass
+from typing import Dict
+
+from tbp.monty.frameworks.config_utils.config_args import Dataclass
+from tbp.monty.frameworks.config_utils.make_dataset_configs import (
+    InformedEnvironmentDataLoaderEvalArgs,
+    InformedEnvironmentDataLoaderTrainArgs,
+)
+
+
+def to_dict(maybe_dataclass: Dataclass | Dict) -> dict:
```

Comment:
Do we not have this method somewhere else already? I feel like I've seen it before. Unless you're moving it here from elsewhere and I haven't gotten to that part yet.

Reply:
I haven't seen it anywhere. There's a bunch of this in configs, which might make `Dataclass | Dict` look familiar?
```python
    motor_system_args: Union[Dict, Dataclass] = field(
        default_factory=lambda: dict(
            policy_class=InformedPolicy,
            policy_args=make_informed_policy_config(
                action_space_type="distant_agent_no_translation",
                action_sampler_class=ConstantSampler,
                rotation_degrees=5.0,
                file_name=Path(__file__).parent / "resources/fixed_test_actions.jsonl",
            ),
        )
    )
```

Reply:
Maybe you were thinking of `config_to_dict` in `src/tbp/monty/frameworks/utils/dataclass_utils.py`? But yeah it's a bit different, otherwise I don't recall anything.